tests/models/grounding_dino/test_modeling_grounding_dino.py

```diff
@@ -741,3 +780,53 @@ def test_cross_attention_mask(self):
         self.assertTrue(torch.allclose(outputs1.logits, outputs_batched.logits[:1], atol=1e-3))
         # For some reason 12 elements are > 1e-3, but the rest are fine
         self.assertTrue(torch.allclose(outputs2.logits, outputs_batched.logits[1:], atol=1.8e-3))
+
+    def test_grounding_dino_loss(self):
+        ds = load_dataset("EduardoPacheco/aquarium-sample", split="train")
```

Comment:
@EduardoPach I pushed copy of the dataset to `hf-internal-testing` org

```suggestion
        ds = load_dataset("hf-internal-testing/tiny-object-detection-aquarium-dataset", split="train")
```